source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3557,7 +3557,6 @@ export abstract class Input extends CardElement implements IInput {
             this._renderedInputControlElement.style.minWidth = "0px";
 
             if (this.isNullable && this.isRequired) {
-                this._renderedInputControlElement.setAttribute("aria-required", "true");
```

Comment:
Does this still announce that it's required?

Reply:
narrator reads it the same way :)